echopop/extensions/inversion/invert.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Callable, Dict, Literal, Union
+
+import pandas as pd
+from numpy.typing import ArrayLike
+
+
+def normalize_scattering_model_parameters(
```

Comment:
I think this can go into `optimize.py` since the output of the optimizer is doing the "inversion" so can just be packaged there.